internal/manager/vars.go

```diff
@@ -27,6 +25,5 @@ var (
 // Controller Manager - Configuration Vars
 // -----------------------------------------------------------------------------
 
-// alwaysEnabled is a convenience alias to indicate that a component is not configurable
-// and is always enabled in any configuration.
-var alwaysEnabled = util.EnablementStatusEnabled
+// neverDisabled is a convenience alias to indicate that a controller cannot be disabled
+var neverDisabled = false
```

Comment:
(nit)

maybe just inline the literal `bool` value?

also, my comment above exposes my stance on double negation :unamused: :sweat_smile:

Reply:
Part of 7c34d87d